src/routes/profile.ts

```diff
@@ -0,0 +1,13 @@
+//Router for everything that relates to the currently connected user
+
+import { Request, Response, Router } from 'express';
+import userController = require('../controllers/user.controller');
+const profileRouter = Router();
+
+profileRouter.get('/', async (req: Request, res: Response) => {
+  res
```

Comment:
You must check if exists but in this case with key = 1 it will be ok always but in the future. You need to verify :)

Reply:
Since this route is used to get the application of the connected user, it will surely exists but it's worth checking. I won't forget to change this 😄